crates/holochain/src/core/workflow/sys_validation_workflow.rs

```diff
@@ -1,4 +1,77 @@
-//! The workflow and queue consumer for sys validation
+//! ### The sys validation workflow
+//!
+//! This workflow runs against all [`Action`]s that are going into the DHT database. Either coming from the authored database or from other nodes on the network.
```

Comment:
```suggestion
//! This workflow runs against all [`DhtOps`]s that are going into the DHT database, either coming from the authored database or from other nodes on the network via gossip and publishing.
```

Reply:
I thought of this too, but even though the workflow runs against all DhtOps, they already are in the DHT database. Actions are created from the DhtOps, but I don't know when exactly that happens.
```suggestion
//! This workflow runs against all [`DhtOp`]s that are in the DHT database, either coming from the authored database or from other nodes on the network.
```

Reply:
Nice, accepted manually to merge with another suggestion